### PR TITLE
add debug logging to entitywhitelist

### PR DESCRIPTION
**Pull request by MilonPL**
## About the PR

title

## Why / Balance

image

### CODE REVIEW

File: Content.Shared/Whitelist/EntityWhitelistSystem.cs [hunk 1]
```diff
@@ -11,12 +12,14 @@ public sealed class EntityWhitelistSystem : EntitySystem
     [Dependency] private readonly SharedRoleSystem _roles = default!;
     [Dependency] private readonly TagSystem _tag = default!;
 
+    private ISawmill _sawmill = default!; // DeltaV
```
Comment (deltanedas) [inline Content.Shared/Whitelist/EntityWhitelistSystem.cs]: its a system just use Log.Debug

Comment (MilonPL) [reply Content.Shared/Whitelist/EntityWhitelistSystem.cs]: won't Log.Debug kill performance when doing a ton of terminal IO? i'm not sure if it's actually more expensive compared to a single sawmill, but that is what's been happening whenever we got invalid component or tag names

Comment (MilonPL) [reply Content.Shared/Whitelist/EntityWhitelistSystem.cs]: update: it kills TPS just as much :trollface:

File: Content.Shared/Whitelist/EntityWhitelistSystem.cs [hunk 2]
```diff
@@ -113,6 +114,20 @@ public bool IsWhitelistPass(EntityWhitelist? whitelist, EntityUid uid)
         if (whitelist == null)
             return false;
 
+        // Begin DeltaV
+        Log.Debug($"Checking whitelist pass for entity {ToPrettyString(uid)}");
```
Comment (deltanedas) [inline Content.Shared/Whitelist/EntityWhitelistSystem.cs]: redundant because of the log right after it

Comment (MilonPL) [reply Content.Shared/Whitelist/EntityWhitelistSystem.cs]: real

**Review by deltanedas — APPROVED** [review]

**Pull request merged**